mithril-client-cli/src/main.rs

```diff
@@ -139,6 +139,27 @@ impl Args {
 
         Ok(Logger::root(Arc::new(drain), slog::o!()))
     }
+
+    fn parse_with_decorator(
+        decorator: &dyn Fn(Result<Self, clap::error::Error>) -> Result<Self, clap::error::Error>,
```

Comment:
Can we use a `StdResult` instead?
Also can we alias `clap::error::Error` with `ClapError`?

Reply:
We have made the ClapError alias.
We keep clap Error for the decorator to have the same signature of the decorated function (try_parse).